### PR TITLE
Blog updates

### PR DESCRIPTION
**Pull request by tibor-mach**
I updated the blogpost to include the Unstructured API (I also keep the FOSS one). Here it is in a markdown version, I will then convert it to medium after the final review.

I also added some info about the SaaS offering of DataChain and a call to action cc @jendefig  (not sure if `support@iterative.ai` is the best place to direct inquiries but I guess it works, let me know if I should change it to something else).

@shcheklein w.r.t. the code consolidation - I will move the (FOSS) script example to the datachain repo after this is merged.

### CODE REVIEW

File: llm/pdf-processing/pdf-processing-post-api.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+Now, we have a versioned dataset with a new version automatically created by DataChain whenever we run the script above. By default, versions are numbered and we can always recall a particular version by specifying it as in this call:
+
+```python
+DataChain.from_dataset("embedded-documents", version=<version number>).show()
+
+```
+
+In the SaaS version of DataChain we also provide lineage tracking for datasets, more robust dataset versioning and auditability, a managed on-demand compute including with GPU clusters to really scale up your workloads and a graphical user interface among other features.
+
+DataChain SaaS is currently available as a private preview in our [DVC Studio](https://studio.dvc.ai/) platform right now, feel free to ask us about that at `support@iterative.ai`!
```
Comment (jendefig) [inline llm/pdf-processing/pdf-processing-post-api.md:200]: I think the correct link is: https://studio.dvc.ai/auth/sign-up

But I'm honestly not sure and that one will get them there too if they click on sign up.  I don't see a waiting list link - only a sign-up for Studio link that says 'Contact Us' which doesn't even really make sense.

File: llm/pdf-processing/pdf-processing-api.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+# Define the output as a DataModel class
+class Chunk(DataModel):
+    key: str
+    text: str
+    embeddings: list[float]
+
+
+# Define embedding encoder
+
+embedding_encoder = HuggingFaceEmbeddingEncoder(config=HuggingFaceEmbeddingConfig())
```
Comment (shcheklein) [inline llm/pdf-processing/pdf-processing-api.py:28]: btw, minor question - do you have to pass an empty `HuggingFaceEmbeddingConfig`?

**Review by jendefig — APPROVED** [review]
Hard for me to really review like this.  And don't know how to look at it how it will really be formatted to be sure.  But made some suggestions. Otherwise LGTM!

File: llm/pdf-processing/pdf-processing-post-api.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+embedding_encoder = HuggingFaceEmbeddingEncoder(
+     config=HuggingFaceEmbeddingConfig()
+)
+
+# Use signatures to define UDF input/output (these can be pydantic model or regular Python types)
+def process_pdf(file: File) -> Iterator[Chunk]:
+    # Ingest the file
+    with file.open() as f:
+        
+        chunks = partition_via_api(
```
Comment (shcheklein) [inline llm/pdf-processing/pdf-processing-post-api.md:76]: let's show as a separate snippet an OS alternative to this?

File: llm/pdf-processing/pdf-processing-post-api.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+    DataChain.from_storage("gs://datachain-demo/neurips")
+    .settings(parallel=-1)
+    .filter(C.file.path.glob("*/1987/*.pdf"))
+    .gen(document=process_pdf)
+)
+```
+
+Let's unpack:
+
+The `from_sto60eding things up once again.
```
Comment (shcheklein) [inline llm/pdf-processing/pdf-processing-post-api.md:135]: typo?

**Review by shcheklein — APPROVED** [review]
seems good to me, see some possible improvements, typos, etc in the comments

**Pull request merged**